cluster/manifests/skipper/service-internal.yaml

```diff
@@ -14,10 +14,9 @@ metadata:
     component: ingress
 spec:
   type: ClusterIP
-{{- if ne .Cluster.Provider "zalando-eks"}}
-# TODO: how to do internal-ingress?
-# function to derive IP from range?
-# Can be hardcoded for ipv4, must be dynamic for ipv6
+{{- if and (eq .Cluster.Provider "zalando-eks") (eq .Cluster.ConfigItems.eks_ip_family "ipv6") }}
+  clusterIP: {{ addressNFromIPV6CIDR .Cluster.ConfigItems.service_ipv6_cidr 50 }}
```

Comment:
Why `50`? I'm assuming based on how the template function works that we're going to reserve the 50th IP address of the v6 CIDR in each  subnet for internal ingress?

And I suppose 50 is an arbitrary number we chose? 😅

Reply:
Will this also somehow help us know if an ingress is internal; i.e. having the 50th address in the subnet will tell us this, or is that not the intention here?

Reply:
Okay I think the (3) part of the PR description explains it, we need this static setup to make local DNS work.

Reply:
50 is a random number, it just has to be not right in the beginning of the range to avoid overlapping with other services that may be assigned an IP automatically when created before this one. We could also use 42 or 131 😂

Reply:
The IP is the one you will get when you resolve `*.ingress.cluster.local` addresses inside the cluster and the IP will point to skipper.

Reply:
Got it. Thanks 🙂